lldb/source/Plugins/Process/Linux/NativeRegisterContextLinux_loongarch64.cpp

```diff
@@ -58,10 +59,21 @@ NativeRegisterContextLinux_loongarch64::NativeRegisterContextLinux_loongarch64(
     std::unique_ptr<RegisterInfoPOSIX_loongarch64> register_info_up)
     : NativeRegisterContextRegisterInfo(native_thread,
                                         register_info_up.release()),
-      NativeRegisterContextLinux(native_thread) {
+      NativeRegisterContextLinux(native_thread),
+      NativeRegisterContextDBReg_arm64(/*CTRL_PLV3_ENABLE=*/0x10U) {
```

Comment:
I like adding these comments but really it should be the name of the parameter in the first part. You could do:
```
/*whatever_the_parameter_name_is=*/0x10U /*CTRL_PLV3_ENABLE/*
```
This is a bit more useful and also any tool like clang-tidy that checks these comments won't complain that `CTRL_PLV3_ENABLE` isn't a prameter.

Reply:
Thanks, I will add parameter annotations. And sorry for the typo here(arm64->loongarch).